active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+
+## PipsPager Theme Resources
+
+You can modify the look of a PipsPager by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+| Resource key | Description | Type
+|:-:|:--| :-- |
+| PipsPagerButtonWidth | Sets the bounding box width for each pip | Double |
+| PipsPagerButtonHeight | Sets the bounding box height for each pip | Double |
+| PipsPagerButtonBorderThickness | Sets the border thickness for each pip | Thickness |
+| PipsPagerNavigationButtonBorderThickness | Sets the border thickness for each navigational button | Thickness |
+| PipsPagerSelectedGlyph | Sets the pip glyph when selected in MDL2 icon set | String |
+| PipsPagerDefaultGlyph | Sets the default pip glyph in MDL2 icon set | String |
+| PipsPagerNavigationButtonHeight | Sets the height for each navigational button | Double |
+| PipsPagerNavigationWidth | Sets the width for each navigational button | Double |
+| PipsPagerNavigationButtonFontSize | Sets the font size for each navigational button | Double |
+| PipsPagerSelectedGlyphFontSize | Sets the size in pixels of the pip glyph when selected | Double |
+| PipsPagerDefaultGlyphFontSize | Sets the size in pixels of the pip glyph at rest| Double |
+
+## PipsPagerButtonVisibility enum
+
+Defines constants that specify how a the buttons of a PipsPager are to be displayed.
+
+| Value | Description |
+|:---:|:---|
+| Visible | The button is visible and enabled, except it is hidden at the extents. For example, when the current page is the first page, the previous button is hidden. |
+| VisibleOnPointerOver | The behavior is the same as Visible *except* that the button is only Visible when the user is hovering over the paging UI with their cursor. |
+| Collapsed | The button is not visible to the user and does **not** take up layout space. (Default for ) |
+
+## PipsPagerTemplateSettings
+
+Defines properties that you can establish template bindings to if defining a new template for the PipsPager.
+
+| Value | Description |
+|:---:|:---|
+| PipsPagerItems | The collection of items to represent the pips in the PipsPager. |
+
+
+# API Details
+
+```IDL
+enum PipsPagerButtonVisibility
+{
+  Visible,
+  VisibleOnPointerOver,
+  Collapsed, 
+}
+
+runtimeclass PipsPagerSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+unsealed runtimeclass PipsPagerTemplateSettings : Windows.UI.Xaml.DependencyObject
```

Comment:
I noticed in the idl that the PipsPagerTemplateSettings constructor is still public. Is this intentional? I believed we had discussed previously this could be private.

Reply:
Recommend: shouldn't have a public constructor.

Reply:
@beervoley, do we have an issue open for this?